#163418457 User authentication forms should have improved design

vic3king opened the pull request:
#### What does this PR do?
Users authentication forms should have improved UI/UX
#### Description of Task to be completed?
- change color theme
- align divs properly
#### How should this be manually tested?
clone repo and serve up login.html and signup.html
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2238799/stories/163418457
#### Screenshots (if appropriate)
![psin](https://user-images.githubusercontent.com/40821284/51599895-9dd1b600-1ef8-11e9-847c-96872ce4c761.PNG)
![psin2](https://user-images.githubusercontent.com/40821284/51599896-9dd1b600-1ef8-11e9-91fe-263a45e33669.PNG)

#### Questions:

JCanaks commented:
Nice work. The dropdown looks very tiny on my end although the screenshot shows otherwise.
-Reduce the thickness of the lines in the form fields 
- Raise the top margin of the forms and make to make them more centralized
- When i am on the signin page i don't need to see the login option in the nav bar and likewise for signup

vic3king committed:
feature(signup-login): Users authentication forms should have improve d UI/UX

- change color theme
- align divs properly
- reduce border size

[delivers #163418457]
